projects/netgrif-components/src/lib/data-fields/multichoice-field/multichoice-list-field/multichoice-list-field.component.html

```diff
@@ -1,11 +1,12 @@
 <mat-selection-list color="primary"
                     [formControl]="formControlRef"
                     [required]="multichoiceField.behavior.required">
-    <mat-label *ngIf="!showLargeLayout.value">{{multichoiceField.title}}
+    <mat-label class="netgrif-label" *ngIf="!showLargeLayout.value">{{multichoiceField.title}}
         <nc-required-label *ngIf="multichoiceField.behavior.required" [isIn]="true"></nc-required-label>
         <br></mat-label>
-    <mat-list-option *ngFor="let option of multichoiceField.choices" [value]="option.key">
+    <mat-list-option checkboxPosition="before" *ngFor="let option of multichoiceField.choices" [value]="option.key">
         {{option.value}}
     </mat-list-option>
+    <mat-hint *ngIf="multichoiceField.description !== undefined && multichoiceField.description !== '' && !multichoiceField.isInvalid(formControlRef)">{{multichoiceField.description}}</mat-hint>
```

Comment:
exclude white-space description?

Reply:
also if a similar check is performed in multiple components, it can be abstracted as a method of the field class and then just called in the template. This would have the added benefit of being able to override the condition if desired, without the need to duplicate the html

Reply:
why we need to check white space strings ? if they are there , then it should be here, its on developer if he wants it or no